cmd/ebrelayer/replay.go

```diff
@@ -18,90 +21,88 @@ import (
 
 // RunReplayEthereumCmd executes replayEthereumCmd
 func RunReplayEthereumCmd(cmd *cobra.Command, args []string) error {
-	//cliContext, err := client.GetClientQueryContext(cmd)
-	//if err != nil {
-	//	return err
-	//}
-	//
-	//// Load the validator's Ethereum private key from environment variables
-	//privateKey, err := txs.LoadPrivateKey()
-	//if err != nil {
-	//	return errors.Errorf("invalid [ETHEREUM_PRIVATE_KEY] environment variable")
-	//}
-	//
-	//// Parse flag --chain-id
-	//chainID := viper.GetString(flags.FlagChainID)
-	//if strings.TrimSpace(chainID) == "" {
-	//	return errors.Errorf("Must specify a 'chain-id'")
-	//}
-	//
-	//// Parse flag --rpc-url
-	//rpcURL := viper.GetString(FlagRPCURL)
-	//if rpcURL != "" {
-	//	_, err := url.Parse(rpcURL)
-	//	if rpcURL != "" && err != nil {
-	//		return errors.Wrapf(err, "invalid RPC URL: %v", rpcURL)
-	//	}
-	//}
-	//
-	//// if !relayer.IsWebsocketURL(args[0]) {
-	//// 	return errors.Errorf("invalid [web3-provider]: %s", args[0])
-	//// }
-	//
-	//tendermintNode := args[0]
-	//web3Provider := args[1]
-	//
-	//if !common.IsHexAddress(args[2]) {
-	//	return errors.Errorf("invalid [bridge-registry-contract-address]: %s", args[1])
-	//}
-	//contractAddress := common.HexToAddress(args[2])
-	//
-	//if len(strings.Trim(args[3], "")) == 0 {
-	//	return errors.Errorf("invalid [validator-moniker]: %s", args[2])
-	//}
-	//validatorMoniker := args[3]
+	cliContext, err := client.GetClientTxContext(cmd)
+
+	if err != nil {
+		return err
+	}
+
+	// Load the validator's Ethereum private key from environment variables
+	privateKey, err := txs.LoadPrivateKey()
+	if err != nil {
+		return errors.Errorf("invalid [ETHEREUM_PRIVATE_KEY] environment variable")
+	}
+
+	// Parse flag --chain-id
+	chainID := viper.GetString(flags.FlagChainID)
+	if strings.TrimSpace(chainID) == "" {
+		return errors.Errorf("Must specify a 'chain-id'")
+	}
+
+	// Parse flag --rpc-url
+	rpcURL := viper.GetString(FlagRPCURL)
```

Comment:
How does this interact with --node param and ternmintNode = args[0] defined below? Do we need / want more than one way to specify?

Reply:
I removed the rpc-url flag completely.  Maybe it was useful before 42?